.gitattributes

```diff
@@ -2,7 +2,7 @@
 *.zip -text diff
 Cargo.lock linguist-generated=false
 pixi.lock linguist-generated=true
-
 # git-lfs stuff
 **/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
 tests/assets/video/*.mp4 filter=lfs diff=lfs merge=lfs -text
+*.rrd filter=lfs diff=lfs merge=lfs -text
```

Comment:
I'd feel better if this was `tests/assets/rrd/*.rrd` instead (or a sub `.gitattributes` file in that folder) :grimacing: